dll/win32/browseui/aclhistory.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * PROJECT:     ReactOS Shell
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Implement CLSID_ACLHistory for auto-completion
+ * COPYRIGHT:   Copyright 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#pragma once
+
+class CACLHistory
+    : public CComCoClass<CACLHistory, &CLSID_ACLHistory>
+    , public CComObjectRootEx<CComMultiThreadModelNoCS>
+    , public IEnumString
```

Comment:
It would be interesting (probably not for this PR) to see if atl has a helper class for IEnum(String), since we are writing a lot of duplicate code now.